<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1131</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="setupPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>COMP3004 Elevator Simulator: Setup Page</string>
      </property>
     </widget>
     <widget class="QLabel" name="floorLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Floors:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="elevatorsInput">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>45</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="elevatorLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Elevators:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="safetyCheckbox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SafetyEvent</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="passengersInput">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>45</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QFrame" name="safetyFrame">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>461</width>
        <height>111</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QRadioButton" name="fireButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>106</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Fire Alarm:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="powerOutButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Power Outage:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="fireInput">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="powerOutInput">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>62</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Event:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>62</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Time:</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="passengerLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Passengers:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>30</y>
        <width>221</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="floorsInput">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>45</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>591</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="passengerEditorFrame"/>
     </widget>
    </widget>
    <widget class="QWidget" name="simulationPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>COMP3004 Elevator Simulator: Simulation Page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resumeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>60</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Resume</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pauseButton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>210</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopButton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>360</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>360</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>424</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Console:</string>
      </property>
     </widget>
     <widget class="QWidget" name="consoleWidget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>861</width>
        <height>431</height>
       </rect>
      </property>
      <widget class="QScrollArea" name="consoleScrollArea">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>861</width>
         <height>431</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>859</width>
          <height>429</height>
         </rect>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>861</width>
           <height>431</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="consoleScrollLayout"/>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
